projects/collection-editor-library/src/lib/components/quml-player/quml-player.component.html

```diff
@@ -1,10 +1,22 @@
  <div class="aspectratio" data-ratio="16:9">
      <div class="content-player">
         <div class="w-100 h-100">
-                <div class="contentViewerIframeShadow">
+                <div class="contentViewerIframeShadow" *ngIf="(editorService?.editorConfig?.config?.renderTaxonomy === undefined)">
```

Comment:
@pradeepkumarcm96 define a variable in component level and check whether undined or not